### ISSUE TITLE
Helium Plasma fuel undervalued in GT5 Large Plasma Turbine

### ISSUE THREAD

**Issue by ghost** [Blood-Asp/GT5-Unofficial#832]
GT4: 1000L Deuterium + 1000L Tritium = 1000L Helium-4 in 128 ticks (6.4 seconds)
GT5: 125L Deuterium + 125L Tritium = 125L Helium-4 in 16 ticks (0.8 seconds)

GT5 Fusion Reactor runs 8x faster.
GT5 Fusion Reactor outputs 8x less fuel.
This is expected.

In GT4: 

Every 128 ticks, 1000L of Helium-4 Plasma is fed into 32 Plasma Generators.
Each plasma generator burns fuel at 10L/s for 2000 ticks (100 seconds), and generates 4096000 EU @ 2048 EU/t.

This yields 65536 EU/t
Minus the production cost of Electrolized Water, Hydrogen, Deuterium, and Tritium, this yields approximately 53K EU/t in net gain.

In GT5:
Every 16 ticks, 125L of Helium Plasma from a single reactor is fed through a Large Plasma Turbine at 7L/t (8L/t would exceed its maximum production capacity) and only 17K EU/t is produced.

There are no rotor materials that provide 65536 EU/t @ 125L/s flow rate to match the fuel value, burnup rate, and energy production of GT4.

So, I suggest:
1) Re-implementing the legacy EV Plasma Generator, which burns Helium-4 Plasma at 10L/s for 100 seconds and generates 4096000 EU @ 2048EU/t

OR

2) Providing a dedicated turbine for Helium-4 Plasma which has an optimal flow rate of 125L/s and produces exactly 65536 EU/t, ignoring any existing turbine math and using GT4 values.

TL;DR Helium Plasma is fine, it's just the god damn turbines that undervalue it because of material stats that were never thought through when Fusion was re-added.

**Comment by codewarrior0** [comment]
>Every 128 ticks, 1000L of Helium-4 Plasma
>Each plasma generator burns fuel at 10L/s

Doesn't this come out to ~8~ 16 Plasma Generators producing ~16k~ 32k EU/t?

>a Large Plasma Turbine at 7L/t (8L/t would exceed its maximum production capacity) and only 17K EU/t is produced.

Please qualify this with the size and material of the turbine rotor. 7L/t of <sup>4</sup>He in a Large Plasma Gen will produce 28k EU/t *before the rotor's efficiency bonus*. I remember using something like a Medium Titanium rotor with a 15% efficiency bonus for just under 33k EU/t total (I had to install a ZPM dynamo hatch to make sure it didn't explode).

<sub>And the extra 0.8125 L/t that the turbine doesn't consume can be routed to a small Plasma Generator using some combination of covers and redstone signaling.</sub>

**Comment by ghost** [comment]
In GT4, compensating for transit time via pipes and delay, one was able to sustain ~31.25 Plasma Generators. The last one in a row would sometimes shut off until a free bucket was inserted. Even with the most efficient pipe routing of plasma, that last one only ran about 25% of the time. Which is still 62K (31x2 + whatever % of 2048 from that 32nd you could get on uptime)

I was using a small HSS-E which after efficency yielded ~21K (17K before the bonus). Checked with scanner + transformer flower & eu-reader to read both GT and IC2 values (just to be sure). less than 100EU difference, which is acceptable given loss & conversion from enet to enet.

The fact remains: A single Fusion Reactor in GT4 paired with 32 plasma generators operating at 99% efficiency (that last one in the row LUL) should yield 62K at the minimum, 64K at the max. Again paying for production would yield 50-53K depending on how good you were with wiring.

None of the turbines in GT5 come even close to matching GT4's values.

**Comment by codewarrior0** [comment]
>In GT4, compensating for transit time via pipes and delay, one was able to sustain ~31.25 Plasma Generators. 

The fuel value for 4He plasma is 4096 EU/L in GT5. It was also 4096 EU/L in GT4, correct? The Fusion Reactor in either GT version produces 7.8125 L/t of 4He. This has a fuel value of exactly 32,000 EU/t. 

How were you able to produce 65,536 EU/t from the GT4 plasma generators? Are the old plasma generators 204.8% efficient?

**Comment by codewarrior0** [comment]
Decompiling the GT 408s jar shows that the fuel value of plasma fuels is 1024 times the plasma's atomic weight. So the fuel value of Helium plasma was nominally 4096 EU/L. I wonder why everybody was getting twice as much, maybe it was a bug in GT4?

**Comment by Blood-Asp** [comment]
I already lowered the processing costs of deuterium/tritium by 2/3 compared to GT5.07.07. The fusion fuel values are untouched.

Also GT4 and GT5 fusion can not be directly compared since GT5 had multiple tiers and fuel types. So it is naturally that the first tiers beginn lower. The final tier of fusion power generation can after all reach 500k+EU/t with little processing cost.

**Comment by codewarrior0** [comment]
The last time I did the math for helium plasma in GT5U, I came up with something like a cost of 13k EU to produce a cell of plasma worth 32k EU. Excluding the reactor's startup cost. It didn't seem all that outrageous.
